Sources/ManifestSerializer/JSONSerialization.swift

```diff
@@ -20,8 +20,14 @@ public protocol JSONSerializable {
 
 public func jsonString(package: PackageDescription.Package) throws -> String {
     
+    #if os(Linux)
+        let options: NSJSONWritingOptions = .PrettyPrinted
+    #else
+        let options: NSJSONWritingOptions = .prettyPrinted
+    #endif
```

Comment:
😭 It'll get fixed once Swift Foundation catches up with the new naming conventions. But yeah.